.htaccess

```diff
@@ -3,6 +3,6 @@ RewriteEngine On
 RewriteCond %{HTTPS} !=on
 RewriteRule ^(.*)$ https://www.bitbybitcoding.org/$1 [R,L]
 
-# remove .php
-RewriteCond %{REQUEST_FILENAME}.php -f
-RewriteRule !.*\.php$ %{REQUEST_FILENAME}.php [L]
\ No newline at end of file
+# remove .html
+RewriteCond %{REQUEST_FILENAME}.html -f
```

Comment:
this file doesnt do anything on github pages.

also, why do we need to rewrite links? IMO thats just going to lead to more 404 errors relating to broken links